saleor/product/filters.py

```diff
@@ -0,0 +1,55 @@
+from collections import OrderedDict
```

Comment:
Missing `from __future__ import unicode_literals`

Reply:
and done